test/transform/FirstParagraphRelocationTransform.test.js

```diff
@@ -0,0 +1,157 @@
+import assert from 'assert'
+import domino from 'domino'
+import fixtureIO from '../utilities/FixtureIO'
+import pagelib from '../../build/wikimedia-page-library-transform'
+
+const moveFirstGoodParagraphUp = pagelib.FirstParagraphRelocationTransform.moveFirstGoodParagraphUp
+const isParagraphGood = pagelib.FirstParagraphRelocationTransform.test.isParagraphGood
+const getElementsToMove = pagelib.FirstParagraphRelocationTransform.test.getElementsToMove
+const getFirstGoodParagraph =
+  pagelib.FirstParagraphRelocationTransform.test.getFirstGoodParagraph
+
+describe('FirstParagraphRelocationTransform', () => {
+  describe('isParagraphGood()', () => {
+    const document = domino.createDocument(`<p id="p1"></p><p id="p2">This p has a bunch of
```

Comment:
Consider moving this variable into the tests its used by since it's only in the first two tests but not the others? Moreover, only the second element is used in the first test and only the first element is used in the second test. Finally, there's no mutations but if one did happen, it would impact all the tests. Additionally, consider trimming the text some so the fixture can fit on a single line.